server/viame_server/event.py

```diff
@@ -10,16 +10,20 @@ def check_existing_annotations(event):
     to no-copy import data
     """
     info = event.info
+    objectType = info.get("type")
+    importPath = info.get("importPath")
 
-    if "annotations.csv" in info["importPath"]:
+    if not importPath or not objectType or objectType != "item":
+        return
+
+    if csvRegex.search(importPath):
+        # Update file metadata
         item = Item().findOne({"_id": info["id"]})
```

Comment:
What if this is a folder named `folder.csv`?  Should there be a `_modelType` check here?